sdk/src/main/java/com/meniga/sdk/models/upcoming/MenigaUpcoming.java

```diff
@@ -488,4 +489,12 @@ public void onFinished(MenigaUpcoming result, boolean failed) {
 	public Result<Void> delete(boolean deleteSeries) {
 		return MenigaUpcoming.apiOperator.deleteUpcoming(this, deleteSeries);
 	}
+
+	public Result<Void> reconcile(ReconcileEntityType entityType, long entityId) {
+		return MenigaUpcoming.reconcile(id, entityType, entityId);
+	}
+
+	public static Result<Void> reconcile(long upcomingId, ReconcileEntityType entityType, long entityId) {
```

Comment:
We could have integration tests for these two methods.

Reply:
Added